vowpalwabbit/multilabel.cc

```diff
@@ -5,13 +5,14 @@
 #include <cfloat>
 #include "gd.h"
 #include "vw.h"
+#include "example.h"
 
 namespace MULTILABEL
 {
```

Comment:
do we care about the lack of testing reported in this file?

Reply:
We do, but I don't think we should tackle it in this pr.